packages/client/components/MenuItem.tsx

```diff
@@ -27,19 +28,21 @@ interface Props {
   parentRef?: RefObject<HTMLDivElement>
 }
 
-const MenuItemStyles = styled('div')<{isActive: boolean; isDisabled: boolean | undefined}>(
-  ({isActive, isDisabled}) => ({
-    alignItems: 'center',
-    backgroundColor: isActive ? PALETTE.SLATE_200 : undefined,
-    color: isDisabled ? PALETTE.SLATE_600 : PALETTE.SLATE_700,
-    cursor: isDisabled ? 'not-allowed' : 'pointer',
-    display: 'flex',
-    '&:hover,:focus': {
-      backgroundColor: isActive ? PALETTE.SLATE_200 : PALETTE.SLATE_100,
-      outline: 0
-    }
-  })
-)
+const MenuItemStyles = styled('div')<{
+  isActive: boolean
+  isDisabled: boolean | undefined
+  to?: string
```

Comment:
+1 Doesn't seem to be used here

Reply:
This was added to provide type support, that allows passing `to`, to the `MenuItemStyles` element for use in `Menu` componet.

Reply:
This was added to provide type support, that allows passing `to`, to the `MenuItemStyles` element for use in `Menu` component.

Reply:
Looked at this locally - we shouldn't need this here, since `to` is exposed to the parent via `useImperativeRef` and isn't used as part of `MenuItemStyles`.